tests/BranchesTest.php

```diff
@@ -10,26 +10,36 @@ class BranchesTest extends TestCase
     public function testBranches(
         array $expected,
         string $defaultBranch,
-        string $minimumCmsMajor,
         string $githubRepository,
         string $composerJson = '',
-        string $branchesJson = '',
-        string $tagsJson = ''
+        string $branchesJson = '[]',
+        string $tagsJson = '[]'
     ) {
         $expectException = $expected === ['__exception__'];
         if ($expectException) {
             $this->expectException(Exception::class);
         }
-        $actual = branches(
-            $defaultBranch,
-            $minimumCmsMajor,
-            $githubRepository,
-            $composerJson,
-            $branchesJson,
-            $tagsJson
-        );
-        if (!$expectException) {
-            $this->assertSame($expected, $actual);
+        try {
+            if ($composerJson) {
+                file_put_contents('__composer.json', $composerJson);
+            }
+            file_put_contents('__branches.json', $branchesJson);
+            file_put_contents('__tags.json', $tagsJson);
```

Comment:
Putting the files there means we don't have a separate code path for tests than we do for normal execution. This is the same as how we handle composer.json data in the generate-matrix tests.